tls/s2n_alerts.c

```diff
@@ -222,7 +229,13 @@ int s2n_process_alert_fragment(struct s2n_connection *conn)
                 conn->config->cache_delete(conn, conn->config->cache_delete_data, conn->session_id, conn->session_id_len);
             }
 
-            /* All other alerts are treated as fatal errors */
+            /*
+             *= https://tools.ietf.org/rfc/rfc8446#section-6
+             *# Unknown Alert types
+             *# MUST be treated as error alerts.
+             *
```

Comment:
I'm pretty sure you don't have to keep the line breaks, so you can fix them when they're super weird.
```suggestion
             *# Unknown Alert types MUST be treated as error alerts.
             *
```